easybuild/easyconfigs/b/BGC-Bayesian-genomic-clines/BGC-Bayesian-genomic-clines-1.03-gompi-2021a.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'MakeCp'
+
+name = 'BGC-Bayesian-genomic-clines'
+version = '1.03'
+local_commit = '9c8c8cb'
+
+homepage = 'https://github.com/zgompert/BGC-Bayesian-genomic-clines'
+description = """Collection of code for Bayesian genomic cline analyses."""
+
+toolchain = {'name': 'gompi', 'version': '2021a'}
+
+github_account = 'zgompert'
+source_urls = [GITHUB_SOURCE]
+sources = ['%s.tar.gz' % local_commit]
+patches = [('%(name)s-%(version)s_Makefile.patch', 1)]
+checksums = [
+    {'9c8c8cb.tar.gz': '105cd1f545c5d9cd55d965436e8e31934fc77a5981399aa6373acd6e42f73cf1'},
+    {'BGC-Bayesian-genomic-clines-1.03_Makefile.patch':
+     'c32853077658731dba2ac7f4916a67f43c9f7679916d602750a8c70d60d810de'},
+]
+
+dependencies = [('GSL', '2.7'),
+                ('HDF5', '1.10.7')]
```

Comment:
```suggestion
    ('HDF5', '1.10.7')
]
```

Reply:
Bit nitpicky, I know, but I think that's the normal style we use for dependencies.

Reply:
No problem, updated in d17c806713b1471aca6cf219219ec4427465144d